### ISSUE TITLE
No Bookmarks?

### ISSUE THREAD

**Issue by david-rouchet** [marcbachmann/node-html-pdf#271]
I've had a look on the options and did not see how to specify to generate bookmarks based on H1, H2 etc... As a matter of fact, the bookmarks are not generated by default.

Is there a way to do it?

**Comment by jazlopez** [comment]
I am totally looking for such feature if possible.

**Comment by holly-weisser** [comment]
+1

**Comment by david-rouchet** [comment]
Still struggling with this, no answer so far nor solution, can anybody help?

**Comment by jazlopez** [comment]
@david-rouchet as far as I know this package is a wrapper of phantomjs binar and as of today phantom does not support bookmark generation out of the box.

Having said that  I would suggest you to use wkhtmlpdf and make sure to install it with the latest patched qt.  

Per the release notes it looks like it supports bookmarks. 
Scroll down to the section **Outlines**
http://madalgo.au.dk/~jakobt/wkhtmltoxdoc/wkhtmltopdf_0.10.0_rc2-doc.html

Hope the information helps.

**Comment by david-rouchet** [comment]
@jazlopez thank you for the feedback. I know this tool and actually we currently use it, but I wanted to go for a pure JS solution to be portable without compiling or targeting a specific platform

**Comment by ezze** [comment]
Probably, [this one](https://github.com/ariya/phantomjs/issues/13959) is a related issue of Phantom.js.

**Comment by david-rouchet** [comment]
@ezze Indeed, this is likely the case. However, no solution has been provided as far as I know... :-(,
